package com.dearbear.exceptionStudy;

public class ExceptionInfo {
	public static void main(String args[]) {
		try {
			int x = Integer.parseInt("22L");
			System.out.println(x);
		} catch (Exception e) {
			// TODO: handle exception
			e.printStackTrace();
//			e.getMessage();
		}
		finally{
			System.out.println("Programe Over");
		}
	}
	public void	cal(int x, int y) {
		
	}
}

class MyException extends Exception{
	public MyException(String ErrorMessage){
		super(ErrorMessage);
	}
}